diff --git a/node_modules/react-native-tab-view/lib/module/TabBarItem.js b/node_modules/react-native-tab-view/lib/module/TabBarItem.js
index 0982d11..579f8b8 100644
--- a/node_modules/react-native-tab-view/lib/module/TabBarItem.js
+++ b/node_modules/react-native-tab-view/lib/module/TabBarItem.js
@@ -192,7 +192,7 @@ const styles = StyleSheet.create({
     flex: 1,
     alignItems: 'center',
     justifyContent: 'center',
-    padding: 10,
+    // padding: 10,
     minHeight: 48
   },
   badge: {
diff --git a/node_modules/react-native-tab-view/lib/module/TabBarItemLabel.js b/node_modules/react-native-tab-view/lib/module/TabBarItemLabel.js
index ad1b324..b219a4b 100644
--- a/node_modules/react-native-tab-view/lib/module/TabBarItemLabel.js
+++ b/node_modules/react-native-tab-view/lib/module/TabBarItemLabel.js
@@ -25,9 +25,10 @@ TabBarItemLabel.displayName = 'TabBarItemLabel';
 const styles = StyleSheet.create({
   label: {
     margin: 4,
-    fontSize: 14,
-    fontWeight: '500',
-    backgroundColor: 'transparent'
+    fontSize: 12.5,
+    // fontWeight: '500',
+    // backgroundColor: 'transparent',
+    fontFamily: "manrope-regular",
   }
 });
 //# sourceMappingURL=TabBarItemLabel.js.map
\ No newline at end of file
